fullscreen.bs

```diff
@@ -534,18 +582,18 @@ user.
  <code>top</code> is zero.
 </ul>
 
-<p>To <dfn>add</dfn> an <var>element</var> to a
+<p>To <dfn id=concept-top-layer-add>add</dfn> an <var>element</var> to a
```

Comment:
That's not enough to remove the need for the id, and also isn't needed to disambiguate. Is it so that HTML could link to this if it used Bikeshed? No other specs use top layer.